src/php/lib/Grpc/AbstractCall.php

```diff
@@ -154,7 +156,11 @@ protected function _deserializeResponse($value)
         if (is_array($this->deserialize)) {
             list($className, $deserializeFunc) = $this->deserialize;
             $obj = new $className();
-            $obj->$deserializeFunc($value);
+            if (method_exists($obj, $deserializeFunc)) {
```

Comment:
The code generator is passing [<response proto className>, "decode"] as `$this->deserialize` right now. I opt not to change codegen now and just defensively handle it here. Once protobuf 3.3.0 is released, I will update grpc's composer.json to require protobuf 3.3.0+. Then we can change all `decode` to `mergeFromString`.